src/components/DateHeader/index.js

```diff
@@ -0,0 +1,21 @@
+
+
+export const DateHeader = ({ data, setData }) => {
```

Comment:
why component named as DateHeader? where are u using the date in that component? should name just Header or SearchToolbar